data/remote_data_source/actors/build.gradle.kts

```
@@ -2,47 +2,27 @@ import java.io.FileInputStream
 import java.util.Properties
 
 plugins {
-    alias(libs.plugins.android.library)
-    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.novix.android.library)
     alias(libs.plugins.kotlin.serialization)
 }
+
 val localProperties = Properties()
 localProperties.load(FileInputStream(rootProject.file("keys.properties")))
+
 android {
     namespace = "com.sanaa.actors"
-    compileSdk = libs.versions.compileSdk.get().toInt()
-
     defaultConfig {
         val apiKey = localProperties["TMDB_API_KEY"].toString()
         buildConfigField("String", "TMDB_API_KEY", "\"${apiKey.trim()}\"")
-        minSdk = libs.versions.minSdk.get().toInt()
-
-        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
-        consumerProguardFiles("consumer-rules.pro")
-    }
-
-    buildTypes {
-        release {
-            isMinifyEnabled = false
-            proguardFiles(
-                getDefaultProguardFile("proguard-android-optimize.txt"),
-                "proguard-rules.pro"
-            )
-        }
-    }
-    compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_11
-        targetCompatibility = JavaVersion.VERSION_11
-    }
-    kotlinOptions {
-        jvmTarget = "11"
+        buildConfigField("String", "TMDB_URL", "\"https://api.themoviedb.org/3\"")
     }
 
     buildFeatures {
         buildConfig = true
     }
 }
 
+
 dependencies {
     implementation(projects.envConfig)
```

This need to change: 
```suggestion
    implementation(projects.preferences)
```

---

it will be changed after merging to div